src/components/PaginatedVirtualList/index.tsx

```diff
@@ -50,6 +52,14 @@ const PaginatedVirtualList = forwardRef<
       ref: listRef,
     });
 
+    useKeyboardShortcut({ charKey: CharKey.PageEnd }, () => {
```

Comment:
Would it be worthwhile to add end-to-end tests for these?

Reply:
Since we never actually invoke the paginated list in our e2e tests.